.env.example

```diff
@@ -7,11 +7,16 @@
 # API CONFIG
 OPENAI_API_KEY=
 OPENAI_API_MODEL=gpt-3.5-turbo # alternatively, gpt-4, text-davinci-003, etc
+
+# CONTEXT STORAGE CONFIG
+CONTEXT_STORAGE_TYPE=pinecone # pinecone | weaviate
+TASK_STORAGE_NAME=tasks
+## PINECONE
 PINECONE_API_KEY=
 PINECONE_ENVIRONMENT=us-east1-gcp
-
-# TABLE CONFIG
-TABLE_NAME=test-table
+## WEAVIATE
```

Comment:
Seems strange to include env. vars for storage that isn't being used by the user. Thoughts on abstracting this out?